packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -672,9 +672,10 @@ class DropdownMenuItem<T> extends _DropdownMenuItemContainer {
   const DropdownMenuItem({
     Key key,
     this.onTap,
-    this.value,
+    @required this.value,
```

Comment:
Is this change required for this pr? If not, let's leave this out because this is probably another breaking change.